core/src/main/java/com/digitalpebble/stormcrawler/protocol/HttpRobotRulesParser.java

```diff
@@ -122,13 +122,18 @@ public BaseRobotRules getRobotRulesSet(Protocol http, URL url) {
                     http.getProtocolOutput(new URL(url, "/robots.txt").toString(), fetchRobotsMd);
             int code = response.getStatusCode();
             bytesFetched.add(response.getContent() != null ? response.getContent().length : 0);
-            // try one level of redirection ?
-            if (code == 301 || code == 302 || code == 307 || code == 308) {
+
+            // According to RFC9309, the crawler should follow at least 5 consecutive redirects
+            // to get the robots.txt file.
+            int max_num_redirects = 5;
```

Comment:
make it a static final field instead?

Reply:
> make it a static final field instead?

+1 

(otherwise, I would argue for `maxNumRedirects` :-) )